storage/innobase/include/dict0dd.h

```diff
@@ -1222,5 +1222,29 @@ bool dd_tablespace_update_cache(THD *thd);
 @return true if it does. */
 bool dd_is_table_in_encrypted_tablespace(const dict_table_t *table);
 
+/* Sets tablespace's DD encryption flag.
+@param[in] Thread       THD
+@param[in] space_name   name of the space for which DD encryption flag is to be
+set */
+bool dd_set_encryption_flag(THD *thd, const char *space_name);
+
+/* Clears tablespace's DD encryption flag.
+@param[in] Thread       THD
+@param[in] space_name   name of the space for which DD encryption flag is to be
+cleared */
+bool dd_clear_encryption_flag(THD *thd, const char *space_name);
```

Comment:
Remove this function. Dead code

Reply:
is used in keyring encryption. Will be used when it gets merged in.